subresource-loading.bs

```diff
@@ -605,18 +605,16 @@ To <dfn>find a matching web bundle registration</dfn> given [=request=]
 
    1. Let |rule| be |registration|'s [=web bundle registration/rule=].
 
-   2. If |url| doesn't meet a path restriction rule, given |registration|'s
-      [=web bundle registration/fetch entry=]'s [=web bundle fetch
-      entry/source=], then return null.
+   2. Let |allowed path| be the result of [shortening]
+      (https://url.spec.whatwg.org/#shorten-a-urls-path) |registration|'s [=web
+      bundle registration/fetch entry=]'s [=web bundle fetch entry/source=].
 
-      Issue: Clarify path restriction rule in algorithm.
+   3. If |url| doesn't start with |allowed path|, then return null.
```

Comment:
What about `uuid-in-package:` URLs?
In Chromium implementation, we skip step 2 and step 3 if |url|'s scheme is `uuid-in-package`.

Reply:
A good point. Mentioned `uuid-in-package`. Done.